lms/static/sass/elements/_program-card.scss

```diff
@@ -91,18 +91,18 @@
 
     .xseries-icon{
       @include float(right);
-      @include margin-right($baseline*0.25 + px);
+      @include margin-right($baseline*0.25);
       background: url('#{$static-path}/images/icon-sm-xseries-black.png') no-repeat;
       background-color: transparent;
       background-size: 100%;
-      width: ($baseline*0.7 + px);
-      height: ($baseline*0.7 + px);
+      width: ($baseline*0.7);
```

Comment:
Seems kinda funky. Might be worth using a fixed pixel value here. This is an icon after all so the size likely won't change.